docs/index.asciidoc

```diff
@@ -112,6 +113,18 @@ input plugins.
 
 &nbsp;
 
+[id="plugins-{type}s-{plugin}-add_hostname"]
+===== `add_hostname`
+
+  added[5.2.0, Field was added to allow users to not automatically add the `host` field in events.]
+  deprecated[5.2.0, In future versions of this plugin, this setting will be removed, and the 'hosts' field will not be added to events.]
```

Comment:
Can we say `deprecated[5.2.1` if the field will be removed for *future* releases? 
To summarize suggestion, "Added in 5.2.0.  Deprecated in 5.2.1"

Reply:
The intent is to deprecate it immediately - the setting is a stop gap until we make the breaking change that will remove the setting.